var/spack/repos/builtin/packages/psmc/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Psmc(MakefilePackage):
+    """mplementation of the Pairwise Sequentially Markovian Coalescent
+    (PSMC) model"""
+
+    homepage = "https://github.com/lh3/psmc"
+
+    version('2016-1-21', git='https://github.com/lh3/psmc.git', commit='e5f7df5d00bb75ec603ae0beff62c0d7e37640b9')
+
+    def setup_environment(self, spack_env, run_env):
+        run_env.prepend_path('PATH', prefix.bin.utils)
+
+    def install(self, spec, prefix):
+        make()
+        with working_dir('utils'):
+            make()
```

Comment:
The `build` phase already runs `make()` for you. If you need to `make()` in a subdirectory as well, you should override the default `build` phase:
```python
def build(self, spec, prefix):
    make()
    with working_dir('utils'):
        make()
```